### DISCUSSION TITLE
[14 settembre 2023] Esercizio 3

### DISCUSSION THREAD

**Discussion by FeddyLix17** [sapienzastudentsnetwork/psd2223#259] [Esami]
Data l’espressione $\Large f = \overline{a} + bc + \overline{b}\overline{a} + (bc + a\overline{c}\ )\overline{a}$ semplificarla e portarla in forma POS.

Realizzare $\Large f$ con soli operatori NAND e con soli operatori NOR.

<details closed>
    <summary> soluzione A-L </summary>

<div align="center">

![image](https://github.com/user-attachments/assets/b90f3f35-4f42-45ca-994a-351c4a03c901)

</div>
</details>

**Comment by Dieal** [comment]
![IMG_20250108_113816_185](https://github.com/user-attachments/assets/520c1759-a6f0-4307-9e0a-8cd8a67b9087)
Dalla traccia si deduce che non si possono utilizzare porte not, quindi vanno utilizzati i loro corrispettivi circuiti NAND e NOR
